modules/ROOT/pages/_partials/proc_self-signed-certs.adoc

```diff
@@ -40,36 +40,54 @@ The resulting file is located in the `/tmp` directory.
 
 [tabs]
 ====
-// tag::excludeDownstream[]
 Android::
 +
 --
+. Emulators
 .. https://support.google.com/android/answer/2819522?hl=en[Set screen lock] on the mobile device to ensure the certificate can be installed.
+.. Drag and drop the certificate file to the emulator, it should be copied to the `Downloads` folder on the device.
+.. Install the certificate on your device:
++
+Go to Settings > Security & location > Advanced > Encryption & credentials > Install from SD card, you will be prompted to choose a file. Navigate to the `Downloads` folder and you should see the certificate file. Tap on the certificate file and Android system should detect the
+certificate and let you install it. Here is https://support.google.com/nexus/answer/2844832?hl=en[a sample guide for Google Nexus devices].
 
+. Real Devices
+.. https://support.google.com/android/answer/2819522?hl=en[Set screen lock] on the mobile device to ensure the certificate can be installed.
 .. Copy the file to your device using one of the following methods:
 +
 * Email attachment (Recommended) - Simply email the certificate to an address
 accessible from the device and download the attachment.
 * Cloud service - Use a cloud storage service such as Dropbox or Google
 Drive which is accessible from the device, browse to the certificate and
 proceed to download it.
-* USB - Attach the device to the machine hosting
-OpenShift via USB and drag the certificate to a devices file system. Here is an https://support.google.com/nexus/answer/2840804?hl=en[example guide for Google Nexus devices]. It may be different for other devices.
-* link:https://developer.android.com/studio/command-line/adb[Android Debug Bridge^] (adb) - Use the adb push command to push the certificate to device or emulator:
+* USB - Attach the device to the machine via USB and drag the certificate to a devices file system. Here is an https://support.google.com/nexus/answer/2840804?hl=en[example guide for Google Nexus devices]. It may be different for other devices. Or, use the link:https://developer.android.com/studio/command-line/adb[Android Debug Bridge^](adb) push command to push the certificate to the connected device:
```

Comment:
I think the fix is part of this: https://github.com/aerogear/mobile-docs/pull/688#pullrequestreview-236519038